docs/applications/containers/beginners-guide-to-LXD-reverse-proxy/index.md

```diff
@@ -0,0 +1,668 @@
+        lxc config device add proxy myport443 proxy listen=tcp:0.0.0.0:443 connect=tcp:127.0.0.1:443 proxy_protocol=true
+
+    {{< output >}}
+Device myport80 added to proxy
+Device myport443 added to proxy
+{{< /output >}}
+
+    The `lxc config device add` command takes as arguments:
+
+    | Argument | Explanation |
+| ----- | ----- |
+| `proxy` | The name of the container. |
+| `myport80` | A name for this proxy device. |
+| `proxy` | The type of the LXD device (LXD _proxy_ device). |
+| `listen=tcp:0.0.0.0:80` | The proxy device will listen on the host (default) on port 80, protocol TCP, on all interfaces. |
+| `connect=tcp:127.0.0.1:80` | The proxy device will connect to the container on port 80, protocol TCP, on the loopback interface. In previous versions of LXD you could have specified `localhost` here. However, in LXD 3.13 or newer, you can only specify IP addresses. |
+| `proxy_protocol=true` | Request to enable the PROXY protocol so that the reverse proxy will get the originating IP address from the proxy device. |
+
+    {{< note >}}
+If you want to remove a proxy device, you can use `lxc config device remove`. If you want to remove the device above called `myport80` you would do it by running the following command:
+
+    lxc config device remove proxy myport80
+
+Where proxy is the name of the container, and myport80 is the name of the device.
+{{< /note >}}
+
+2.  Start a shell in the `proxy` container.
+
+        lxc exec proxy -- sudo --user ubuntu --login
+
+3.  Update the package list.
+
+        sudo apt update
+
+4.  Install NGINX in the container.
+
+        sudo apt install -y nginx
+
+5.  Logout from the container.
+
+        logout
+
+### Setting up the Apache Web Server in the Reverse Proxy
+
+The reverse proxy container is running and the NGINX package has been installed. To work as a reverse proxy, we add the appropriate website configuration so that NGINX can identify (with `server_name` below) the appropriate hostname, and then pass (with `proxy_pass` below) the connection to the appropriate LXD container.
+
+1.  Start a shell in the `proxy` container.
+
+        lxc exec proxy -- sudo --user ubuntu --login
+
+2.  Create the file `apache1.example.com` in `/etc/nginx/sites-available/` for the configuration of your first website.
+
+    {{< file "apache1.example.com" >}}
+server {
+        listen 80 proxy_protocol;
+        listen [::]:80 proxy_protocol;
+
+        server_name apache1.example.com;
+
+        location / {
+                proxy_set_header Host $host;
+                proxy_set_header X-Real-IP $remote_addr;
+                proxy_pass http://apache1.lxd;
+        }
+
+        real_ip_header proxy_protocol;
+        set_real_ip_from 127.0.0.1;
+}
+{{</ file >}}
+
+    You can run `sudo nano /etc/nginx/sites-available/apache1.example.com` to open up a text editor to add the configuration. Note, in this case you only need to edit the `server_name` to be the hostname of the website.
```

Comment:
I did not need to edit the proxy_pass field and so I removed that from this text.